sources/web/datalab/polymer/notebook.ts

```diff
@@ -29,6 +29,7 @@ const CommandId = {
   ERROR: 'error',
   LOAD_NOTEBOOK: 'load-notebook',
   PICK_PROJECT: 'pick-project',
+  REDIRECT_TO_NOTEBOOK: 'redirect-to-notebook',
```

Comment:
Seems to me like this is a generic redirect functionality, should we call this just `REDIRECT` or `CHANGE_URL`?

Reply:
Yes, we could make it more generic. I wanted to restrict it just to notebooks for now, to make sure the notebook couldn't redirect to something else. We can consider the potential security of issues of that if that use case arises.